src/renderer/components/privacy-settings/privacy-settings.js

```diff
@@ -36,8 +38,15 @@ export default defineComponent({
     rememberHistory: function () {
       return this.$store.getters.getRememberHistory
     },
-    saveWatchedProgress: function () {
-      return this.$store.getters.getSaveWatchedProgress
+    watchedProgressSavingMode() {
+      return this.$store.getters.getWatchedProgressSavingMode
+    },
+    watchedProgressSavingModeValueNamePairs() {
+      return [
+        ['auto', this.$t('Settings.Privacy Settings.Watched Progress Saving Mode.Modes.Auto')],
+        ['semi-auto', this.$t('Settings.Privacy Settings.Watched Progress Saving Mode.Modes.Semi-auto')],
+        ['never', this.$t('Settings.Privacy Settings.Watched Progress Saving Mode.Modes.Never')],
+      ]
```

Comment:
Can you please split this up into two separate array like we do for the other dropdowns. If you want to group them together we can place the two arrays next to each other when we migrate the privacy settings to the composition API.

Reply:
Split